data/data-pipeline/data_pipeline/etl/score/config/datasets.yml

```diff
@@ -0,0 +1,44 @@
+---
+datasets:
+  - long_name: "FEMA National Risk Index"
+    short_name: "nri"
+    module_name: national_risk_index
+    last_updated_year: 2020
+    description: "Dataset from FEMA that identifies communities most at risk to 18 natural hazards."
+    input_geoid_tract_field_name: "TRACTFIPS"
+    load_fields:
+      - short_name: "ex_loss"
+        df_field_name: "RISK_INDEX_EXPECTED_ANNUAL_LOSS_SCORE_FIELD_NAME"
+        long_name: "FEMA Risk Index Expected Annual Loss Score"
+        field_type: float
+        tile_include: true
+        csv_download: true
+        excel_download: true
+      - short_name: "ex_pop_loss"
+        df_field_name: "EXPECTED_POPULATION_LOSS_RATE_FIELD_NAME"
+        long_name: "Expected population loss rate (Natural Hazards Risk Index)"
+        field_type: float
+        tile_include: true
+        csv_download: true
+        excel_download: true
+      - short_name: "ex_ag_loss"
+        df_field_name: "EXPECTED_AGRICULTURE_LOSS_RATE_FIELD_NAME"
+        long_name: "Expected agricultural loss rate (Natural Hazards Risk Index)"
+        field_type: float
+        tile_include: true
+        csv_download: true
+        excel_download: true
+      - short_name: "ex_bldg_loss"
```

Comment:
I feel like we're missing a few fields to configure per data source / fields that would be really helpful, such as: do we want to create percentiles and/or reverse percentiles for this field?

Reply:
Also do we want to include the field in the comparison tool either as 1) an index to compare, or 2) a characteristic to compare (such as tract income).

Reply:
Reverse percentiles code now: data/data-pipeline/data_pipeline/etl/score/etl_score.py:477

Reply:
We discussed with @lucasmbrown-usds and it totally makes sense to add any field metadata to be described on the `YAML` dataset. However anything related to `etl/score` and `etl/score_post` will come in separate PRs.

Reply:
Added `reverse_percentile` optional.